user_docs/en/userGuide.t2t

```diff
@@ -2186,6 +2188,12 @@ This category contains the following options:
 This combo box allows you to choose the language to be used for text recognition.
 To cycle through available languages from anywhere, please assign a custom gesture using the [Input Gestures dialog #InputGestures].
 
+==== Periodically refresh recognized content ====[Win10OcrSettingsAutoRefresh]
+When this check box is enabled, NVDA will automatically refresh the recognized content when a recognizition result has focus.
+This can be very useful when you want to monitor constantly changing content, such as when watching a video with sub titles.

```

Comment:
Fix typos

```suggestion
When this checkbox is enabled, NVDA will automatically refresh the recognized content when a recognition result has focus.
This can be very useful when you want to monitor constantly changing content, such as when watching a video with subtitles.
```